aten/src/ATen/native/DistributionTemplates.h

```diff
@@ -157,47 +158,6 @@ at::Tensor& random_from_to_impl(at::Tensor& self, int64_t from, c10::optional<in
 
 // ==================================================== Normal ========================================================
 
-// This function computes broadcasted size of mean and std, resize the output to the broadcasted size if it was empty
-// [Note] The following features will be deprecated in version 1.6 release and function signature will be changed after
-//   When mean and std are not broadcastable but have same number of elements:
-//     This function will resize the output to the size of mean if it was empty.
-//     This function will reshape the std to the shape of mean.
-//     This function will return true in deprecated case, false in broadcastable case and throw in all other cases before deprecation.
-//     This function will not return and throw if mean and std are not broadcastable after deprecation
-static bool resize_output_for_normal(at::Tensor& output, const at::Tensor& mean, const at::Tensor& std) {
-  bool expandable = at::are_expandable(mean.sizes(), std.sizes());
-  bool empty_output = output.numel() == 0;
-
-  if (expandable) {
-    auto shape = at::infer_size(mean.sizes(), std.sizes());
-    TORCH_CHECK(
```

Comment:
Are we missing this? If i'm understanding correctly, this function currently will error out if you pass in an `output` tensor with the "wrong" size. I don't think that'll happen below when you call `resize_output()`

Reply:
yeah, but isn't it the whole point of this pr and resize_output? this is an intermediate step toward having meta-aware kernels for this op. and with meta you kinda always have the right size by definition (that's just the point of meta kernels -- to set up the output size). or am i missing something?

Reply:
discussed below, marking as resolved